backend/SafeSender.StorageAPI/docker-compose.yml

```diff
@@ -8,22 +8,21 @@ services:
       - "27017:27017"
     volumes:
       - safe-sender-mongodb-data:/data/db
-    external_links:
```

Comment:
Have you ensured that the microservice can use mongodb container by its link (name) without this code?

Reply:
The external_links option is a legacy feature and generally not recommended for use. In most cases, depends_on will handle dependencies correctly. Additionally, containers in the same docker-compose file are in the same network by default, so they can reach each other by service name.